bin/templates/cordova/lib/log.js

```diff
@@ -21,32 +21,23 @@
 
 var path = require('path');
 var os = require('os');
-var Q = require('q');
-var child_process = require('child_process');
+var execa = require('execa');
 var ROOT = path.join(__dirname, '..', '..');
 
 /*
  * Starts running logcat in the shell.
  * Returns a promise.
  */
 module.exports.run = function () {
-    var d = Q.defer();
```

Comment:
The `Q` changes I think should done in the dropping Q PR : https://github.com/apache/cordova-android/pull/833, but this type of work kinda blends together.

Reply:
What I wanted to do in this PR is to ensure that all external process spawning would be done with `execa`. The fact that this also happens to remove `Q` usage is circumstantial. So I think this is alright to have here.

Also note that this PR is supposed to be merged before #833. Maybe then it makes more sense?